src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -475,6 +474,10 @@ class ChargeFeeFine extends React.Component {
     const selectedFeeFine = feefines.find(f => f.id === feeFineTypeId);
     const currentOwnerFeeFineTypes = feefines.filter(f => f.ownerId === resources.activeRecord.ownerId);
     const selectedOwner = owners.find(o => o.id === initialOwnerId);
+    const payments = _.get(resources, ['payments', 'records'], []).filter(
+      p => p.ownerId === this.state.ownerId || resources.activeRecord.ownerId
+    );
+
```

Comment:
Am I misreading or will this include all payments when `resources.activeRecord.ownerId` evaluates as truthy? Is that the intent?

Reply:
Yes, that is right. 
I see this piece is confusing, I guess would be better just to extract all payments here. Since `payments` are passing to `ActionModal`, where it will filter again (because in multiple payment we have opportunity to change owner there as well).
Will do that

Reply:
Ah, OK. I'm fine with either implementation. I see you pushed up a new version already, but if you think this code is better, I'm fine with it. Just include a comment like the one above about how `ActionModal` will filter this. 

Thanks for the explanation!

Reply:
I think better is not to filter `payments` here indeed for several reasons: the clearer code, and  avoid unnecessary process